dev_docs/cxg.md

```diff
@@ -119,46 +116,61 @@ This optional field contains a copy of the category color table, which MAY be us
 
 ## Corpora Schema Encoding
 
-The [Corpora schema](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md) and [Corpora AnnData encoding](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) define a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following shall apply.
+Note: This section is technically not part of the CXG specification, but rather provides an expectation of what metadata will be available in a CXG file that is converted from Corpora Schema-compliant H5AD file; we use "WILL" rather than "MUST" to document these expectations.
+
+The [Corpora Schema 2.0.0](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema) defines a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following WILL apply.   
 
 ### Corpora metadata property
 
-A CXG containing a Corpora dataset will contain a property in the __cxg_group_metadata__ field named `corpora`. The value will be a JSON encoded string, which in turn contains all properties defined in the [Corpora AnnData uns](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md#uns) container. For example:
+A CXG containing a Corpora dataset WILL contain a property in the `__cxg_group_metadata__` field named `corpora`. The value WILL be a JSON encoded string, which in turn contains all properties defined in the [Corpora Schema 2.0.0 "uns"](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema.md#uns-dataset-metadata) AnnData container. The entire encoding WILL be JSON, rather than a hybrid Python/JSON encoding, but will otherwise follow the data structure defined by the Corpora schema.
+
+
+These Corpora metadata properties WILL exist in the CXG `__cxg_group_metadata__` field named `corpora`: 
+* `schema_version`
+* `title`
+* `X_normalization`
+* `X_approximate_distribution`
 
+These Corpora metadata properties MAY exist in the CXG `__cxg_group_metadata__` field named `corpora`:
+* `contributors`
+* `default_embedding`
+* `project_name`
+* `project_description`
+* `project_links`
+* `preprint_doi`
+* `publication_doi`
```

Comment:
none of these are required by Schema 2.0.0, but I'm assuming they may exist by convention alone. As these are the only props that will be passed through into the CXG, I suppose it's worth listing them here.

Reply:
This is schema 1.0.0 which embedded collection (project) information in the dataset prior to the UX. Deprecated in schema 1.1.0.  See https://github.com/chanzuckerberg/single-cell/issues/75.

The remaining 1.0.0 datasets in the corpus (can identify them by the missing cell count in _All Collections_ view will be updated during re-curation by Stanford.

Reply:
Thanks, so I will remove this section. 

Separately, we may want to remove from the H5AD->CXG conversion logic [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/d9c534968d592aa9ecf46791b111e1e6edc061ba/backend/corpora/common/utils/corpora_constants.py#L93). I'm assume that will be handled by later schema 1.x deprecation work..